map2.js

```diff
@@ -31,3 +31,18 @@ var phoneBook = [
 // For example, the 0th element of the new array should be {Abe: "1-111-111-1111"}
 
 //Your code here
+//var phoneNumber = (Object.values(phoneBook[0])).toString()
+//console.log("1-".concat(phoneNumber))
+
+
+const newPhoneBook = phoneBook.map((contact) => {
+  var phoneNumber = (Object.values(contact)).toString();
+  var newNumber = ("1-".concat(phoneNumber))
+  var name =(Object.keys(contact)).toString();
+  
+   return {name, newNumber}
```

Comment:
you're *sooooo* close here! using a comma makes the object think that the name of the variable is the key, and the value of the variable is the value. If you want the value of `name` to be the key, and the value of `newNumber` to be the value, set up the two variables to look like a key-value pair, like so: `return {name: newNumber}` Nice work thinking through the hard stuff, though! 👏